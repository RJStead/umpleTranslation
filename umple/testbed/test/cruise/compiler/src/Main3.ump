generate Java "../src-gen-umple";

class Main3
{
    void main(String [] argv){
        int a = 1;
        b = a;
    }
}
