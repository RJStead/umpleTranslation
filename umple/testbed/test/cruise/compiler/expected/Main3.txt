Main3.ump:7: error: cannot find symbol
        b = a;
        ^
  symbol:   variable b
  location: class Main3
1 error
